cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -32,7 +32,8 @@ spec:
             - --sink=gcl
             - --sink=gcmautoscaling
```

Comment:
Will do it. Alternatively we use googleinfluxdb monitoring level in our autoscaling e2e tests.